platform/node/test/ignores.json

```diff
@@ -40,6 +36,7 @@
   "render-tests/line-join/property-function": "https://github.com/mapbox/mapbox-gl-js/pull/5020",
   "render-tests/line-join/property-function-dasharray": "https://github.com/mapbox/mapbox-gl-js/pull/5020",
   "render-tests/regressions/mapbox-gl-js#2305": "https://github.com/mapbox/mapbox-gl-native/issues/6927",
+  "render-tests/regressions/mapbox-gl-js#2929": "https://github.com/mapbox/mapbox-gl-native/issues/9704",
```

Comment:
https://github.com/mapbox/mapbox-gl-native/issues/9704 is closed -- is this ignore still necessary?

Reply:
You're right - this is no longer necessary :+1: removing